File: ios/Plugin/CleverPushPlugin.swift
import Foundation

@objc public class CleverPushPlugin: NSObject {
    
    @objc public func getSubscriptionId(_ value: String) -> String {
        return value
    }
    
    @objc public func isSubscribed(_ value: Bool) -> Bool {
        return value
    }
    
    @objc public func unsubscribe () {}
    
    @objc public func subscribe () {}
    
    @objc public func enableDevelopmentMode() {}
    
    @objc public func initCleverPush() {}
    
    @objc public func showTopicsDialog() {}
    
}

File: android/src/main/java/com/cleverpush/plugins/capacitor/CleverPushPlugin.java
package com.cleverpush.plugins.capacitor;

public class CleverPushPlugin {

    public String getSubscriptionId(String value) {
        System.out.println(value);
        return value;
    }

    public boolean isSubscribed(boolean value) {
        System.out.println(value);
        return value;
    }
    public void unsubscribe () {}
    public void subscribe () {}
    public void enableDevelopmentMode() {}

    public void initCleverPush() {

    }
    public void showTopicsDialog() {}

}
